app/off.py

```diff
@@ -0,0 +1,22 @@
+import requests
+
+
+def lastEdit(url, query):
+    """
+    Helper function to return data for last-edits
+    """
+    search_url = f"{url}/api/v2/search"
+    response_API = requests.get(search_url, params=query)
+    data = response_API.json()
+    counts = data["count"]
+    tags = data["products"]
+
+    html = "\n".join(
+        f'<li>{tag["product_name"]} ({tag["code"]}) edited by {tag["last_editor"]} on {tag["last_edit_dates_tags"][0]}</li>'
+        for tag in tags[0:10]
+        if "product_name" in tag
+    )
```

Comment:
design pattern for i18n (not to do now)

```suggestion
    html = []
    for tag in tags[0:10]:
        html.append('<li>')
        html.append(
            f'{tag["product_name"] or ''} ({tag["code"]}) edited by {tag["last_editor"]} on {tag["last_edit_dates_tags"][0]}'
        )
        html.append('</li>')
    ...
    html = [
      "<ul><p>",
      f"Total number of edits {counts}"
      "</p>",
    ] + html + [
        "</ul>"
    ]
    ....
    text = "".join(html)
```